libs/core/src/lib/sources.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { Subject } from 'rxjs';
+import { Effect } from './effect';
+import { setup } from './test-helper';
+
+it('sources data from subject', () => {
+  const value = { count$: new Subject() };
+  const effect: Effect<typeof value> = ({ sources }) => {
+    return sources.count$();
+  };
+  setup(
+    { value, effect },
+    ({ expectObservable, effect$, hot, storeEvent$ }) => {
+      hot('a').subscribe(value.count$);
```

Comment:
@kylecannon any idea why changing this to cold breaks the test? It should be no different, in both cases we subscribe synchronously before the rest of the test runs.